### DISCUSSION TITLE
Image component doesn't work with Google profile images urls

### DISCUSSION THREAD

**Discussion by zanzlender** [vercel/next.js#58961] [Help] [answered]
### Summary

Hello, I've been experiencing an issue with this which I could not figure out why... When trying to add Google's profile image storage as a remote pattern it always says it's not configured when using it.

My next-config:

```js
{
  images: {
    remotePatterns: [
      {
         protocol: "https",
         hostname: "*.googleusercontent.com",
         port: "",
         pathname: "*",
      }
    ]
  }
}
```

I also tried with `lh3.googleusercontent.com` which is the full address but it still doesn't work. Also, I am worried that if Google for some reason changes this hostname in the future my app will crash.

Am I doing something wrong or do I have no other option but to use the regular <img> tag?

**Comment by ranefattesingh** [comment, marked as answer]
I fixed it by trying the following setting

`pathname: "**"`

**Reply by tomharrisonjr** [reply]
Next 15.4, didn't work for me
